package.json

```
@@ -143,6 +143,12 @@
 		"generate-client:tldraw-api": "openapi-generator-cli generate -c ./openapitools.json --generator-key tldraw-api",
 		"pregenerate-client:boards-api": "rimraf ./apps/server/src/infra/boards-client/generated",
 		"generate-client:boards-api": "openapi-generator-cli generate -c ./openapitools.json --generator-key boards-api",
+		"pregenerate-client:column-api": "rimraf ./apps/server/src/infra/column-client/generated",
```

consistency with the rest
```suggestion
		"pregenerate-client:columns-api": "rimraf ./apps/server/src/infra/columns-client/generated",
```